example.csv

```diff
@@ -0,0 +1 @@
+FileName_OrigRNA,PathName_OrigRNA,FileName_OrigER,PathName_OrigER,FileName_OrigAGP,PathName_OrigAGP,FileName_OrigMito,PathName_OrigMito,FileName_OrigDNA,PathName_OrigDNA,Metadata_Plate,Metadata_Well,Metadata_Site,Metadata_AbsPositionZ,Metadata_AbsTime,Metadata_BinningX,Metadata_BinningY,Metadata_ChannelID,Metadata_ChannelName,Metadata_Col,Metadata_ExposureTime,Metadata_FieldID,Metadata_ImageResolutionX,Metadata_ImageResolutionY,Metadata_ImageSizeX,Metadata_ImageSizeY,Metadata_MainEmissionWavelength,Metadata_MainExcitationWavelength,Metadata_MaxIntensity,Metadata_ObjectiveMagnification,Metadata_ObjectiveNA,Metadata_PlaneID,Metadata_PositionX,Metadata_PositionY,Metadata_PositionZ,Metadata_Row
```

Comment:
Should this file be at the top level, or in an "examples" or something subfolder?